README.md

```diff
@@ -17,6 +17,47 @@ Run these commands:
     bundle install
     bundle exec rake db:migrate
 
+
+
+Using Vagrant for Development
+-----------------------------
+
+Prerequisites:
+
+* [VirtualBox](https://www.virtualbox.org/) (Version 4.3)
+* [Vagrant](http://www.vagrantup.com/) (Version 1.6)
+* [vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest/) (`vagrant plugin install vagrant-vbguest`)
+* [Ansible](http://www.ansible.com/) (Version 1.7 or greater; [installation instructions](http://docs.ansible.com/intro_installation.html))
+
+
+Add this line to your `/etc/hosts` or equivalent:
+
+    192.168.50.21   heidrun
+
+Then do this:
+
+    $ cd /path/to/this/directory
+    $ vagrant up
+    $ vagrant reload  # Because of o/s packages having been upgraded
+    $ vagrant ssh
+    $ cd /vagrant
+    $ RAILS_ENV=development bundle exec rake jetty:start
```

Comment:
You shouldn't need to declare `RAILS_ENV` explicitly on these lines, or when you run `rake ci`